src/db/models/governingEntity.ts

```diff
@@ -12,3 +15,25 @@ export type GoverningEntityId = Brand<
 export const GOVERNING_ENTITY_ID = brandedType<number, GoverningEntityId>(
   t.number
 );
+
+export default defineIDModel({
```

Comment:
When running the unit tests for `hpc_service`, it fails with the message:

```
Table governingEntity is missing columns in definition: currentVersion, latestVersion, latestTaggedVersion, versionTags
```

```suggestion
export default defineLegacyVersionedModel({
```